### ISSUE TITLE
How to use with multiple gtfs data directories?

### ISSUE THREAD

**Issue by s2t2** [lovehandle/gtfs-orm#1]
I wasn't able to configure the `GTFS.path` or `GTFS::ORM.path` methods, per the instructions.

```
GTFS.path("my/path")
=> NoMethodError: undefined method `path' for GTFS:Module
GTFS::ORM.path = "my/path"
=> "my/path"
GTFS::ORM.path 
=> "./data"
```

I was able to override `GTFS::ORM::DEFAULT_PATH` in my project, which lets me use the ORM for a single gtfs data directory.

``` rb
module GTFS
  module ORM
    DEFAULT_PATH = "my/path"
  end
end
```

But how can I configure different instances of GTFS::ORM to use with different gtfs data directories?

**Comment by ahayworth** [comment]
This would be fixed by https://github.com/rynclsnr/gtfs-orm/pull/2

**Comment by lovehandle** [comment]
Like @ahayworth said - should be fixed by #2.
